frontend/Rhyolite/Frontend/App.hs

```diff
@@ -264,7 +264,7 @@ runPrerenderedRhyoliteWidget
 runPrerenderedRhyoliteWidget url child = do
   rec (notification :: Event t (View app ()), response) <- fmap (bimap (switch . current) (switch . current) . splitDynPure) $
         prerender (return (never, never)) $ do
-          appWebSocket :: AppWebSocket t app <- openWebSocket' url request'' $ fmap (\_ -> ()) <$> nubbedVs
+          appWebSocket :: AppWebSocket t app <- openWebSocket' url request'' $ fmapMaybe (\c -> if c == mempty then Nothing else Just ()) <$> nubbedVs
```

Comment:
`fmapMaybe` is now deprecated no?

Reply:
No, the class FunctorMaybe is deprecated but the function fmapMaybe is not. It's just a synonym for mapMaybe now.